standalone-server/webroot/query-graphs.html

```diff
@@ -25,10 +25,10 @@
     <meta name="msapplication-config" content="icons/browserconfig.xml">
     <meta name="theme-color" content="#ffffff">
     <!-- main style sheet  -->
-    <link href='query-graphs.css' rel='stylesheet' type='text/css'>
+    <link href='query-graphs.bundle.css' rel='stylesheet' type='text/css'>
 </head>
 <body>
 </body>
 <!-- bundle created by browserify -->
-<script src="query-graphs.bundle.js"></script>
+<script src="query-graphs.min.js"></script>
```

Comment:
any impact to debugability?

Reply:
yes, this turned out to be a bit annoying. Reverted it